### DISCUSSION TITLE
Library management

### DISCUSSION THREAD

**Discussion by Omicronlawful** [modular/modular#256] [Q&A]
Ive noticed that there is quite some similarity to rust. Will the library management be similar to rust. Rust is awesome for having a toml where the version is defined and everything happens automatilcy. Python is a nightmare to work with(missing requirements…). So many projects are broken because of this. Is there is similar attempt for mojo and is there some rust integration? Will it be possible to use rust libraries or c++ libraries directly without a wrapper to make it usable?

**Comment by daydun** [comment]
C/C++ interop is on the roadmap: https://docs.modular.com/mojo/roadmap.html#cc-interop
